mdm/orientdb-wrapper/src/main/java/com/gentics/mesh/core/data/dao/impl/SchemaDaoWrapperImpl.java

```diff
@@ -140,8 +148,19 @@ public HibSchemaVersion findVersionByUuid(HibSchema schema, String versionUuid)
 	}
 
 	@Override
-	public Result<? extends HibNodeFieldContainer> findDraftFieldContainers(HibSchemaVersion version, String branchUuid) {
-		return toGraph(version).getDraftFieldContainers(branchUuid);
+	public long findDraftFieldContainerCount(HibSchemaVersion version, String branchUuid) {
```

Comment:
This implementation will be inefficient (for large datasets) and it seems that this is only used to add some metrics. Maybe this method should be removed.